Mathlib/Computability/PartrecCode.lean

```diff
@@ -206,8 +206,6 @@ theorem encode_lt_rfind' (cf) : encode cf < encode (rfind' cf) := by
 
 end Nat.Partrec.Code
 
--- Porting note: Opening `Primrec` inside `namespace Nat.Partrec.Code` causes it to resolve
```

Comment:
I trust that you checked these two notes. (I didn't carefully verify if they are fixed now.)